pyo3-macros-backend/src/pyclass.rs

```diff
@@ -786,6 +803,12 @@ fn impl_simple_enum(
                 use #pyo3_path::conversion::ToPyObject;
                 use #pyo3_path::types::PyAnyMethods;
                 use ::core::result::Result::*;
+                #[deprecated(
+                    since = "0.22.0",
+                    note = "Implicit equality for simple enums in deprecated. Use `#[pyclass(eq)` to opt-in to the new behavior."
```

Comment:
In the note, "in" should be "is".

Reply:
Thanks! I'll fix it up together with the tests, once we agreed if dropping direct integer comparison is fine.